services/thumbnails/pkg/preprocessor/preprocessor.go

```diff
@@ -41,6 +45,28 @@ func (i GifDecoder) Convert(r io.Reader) (interface{}, error) {
 	return img, nil
 }
 
+type AudioDecoder struct{}
+
+func (i AudioDecoder) Convert(r io.Reader) (interface{}, error) {
+	readSeeker := NewLazyReadSeeker(r)
+	m, err := tag.ReadFrom(readSeeker)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Don't use `log.Fatal`, just return the error and let others care about the severity

Reply:
Yeah, debugging leftover oversight. Code is going away anyway 👍